File: 116/[2019-投稿-A]剪刀石頭布.cpp
#include <bits/stdc++.h>
using namespace std;

#define miyuki_is_my_wife ios::sync_with_stdio(false), cin.tie(nullptr)

int main()
{
    miyuki_is_my_wife;

    int a{}, b{};
    cin >> a >> b;
    if(a==5&&b==0)
        cout<<"A wins!!"<<endl;
    else if(a==0&&b==2)
        cout<<"A wins!!"<<endl;
    else if(a==2&&b==5)
        cout<<"A wins!!"<<endl;
    else if(a==5&&b==2)
        cout<<"B wins!!"<<endl;
    else if(a==2&&b==0)
        cout<<"B wins!!"<<endl;
    else if(a==0&&b==5)
        cout<<"B wins!!"<<endl;
    else if(a == b)
        cout<<"Both win!!"<<endl;
    else
        cout<<"Are you kidding me?"<<endl;

    return 0;
}

File: 116/[2019-投稿-A]剪刀石頭布.py
def main():
    a, b = map(int, input().split())
    
    flag = 0
    flag += (a == 5 and b == 0)
    flag += (a == 0 and b == 2)
    flag += (a == 2 and b == 5)
    
    if flag:
        print("A wins!!")
    else:
        if a == b:
            print("Both win!!")
        elif a not in [0, 2, 5] or b not in [0, 2, 5]:
            print("Are youo kidding me?")
        else:
            print("B wins!!")

if __name__ == "__main__":
    # miyuki is my wife and sister

    main()
